pkg/kubectl/cmd/update.go

```diff
@@ -42,7 +42,7 @@ Examples:
 
   $ cat pod.json | kubectl update -f -
   <update a pod based on the json passed into stdin>
-  
+
```

Comment:
This whitespace cleanup is what's causing travis/shippable to complain. Run `hack/run-gendocs.sh`